composer.json

```diff
@@ -45,6 +45,8 @@
         "mglaman/phpstan-drupal": "^1.1",
         "phpstan/extension-installer": "^1.1",
         "phpstan/phpstan-deprecation-rules": "^1.0",
+        "nette/finder": "^2.5",
```

Comment:
Not 100% sure if nette/finder is needed as nothing hasn't been broken before, then again it is used in https://github.com/wunderio/code-quality/blob/main/src/Drupal/DrupalAutoloader.php